fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -468,6 +472,10 @@ private void executeByNereids(TUniqueId queryId) throws Exception {
         context.setStartTime();
         profile.getSummaryProfile().setQueryBeginTime();
         context.setStmtId(STMT_ID_GENERATOR.incrementAndGet());
+
+        if (checkBlockRules()) {
```

Comment:
I think the `checkBlockRules()` no need to return a `bool` value?

Reply:
fixed